packages/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingStoreImpl.java

```diff
@@ -18,8 +18,31 @@
 public class ReactNativeFirebaseMessagingStoreImpl implements ReactNativeFirebaseMessagingStore {
 
   private static final String S_KEY_ALL_NOTIFICATION_IDS = "all_notification_ids";
-  private static final int MAX_SIZE_NOTIFICATIONS = 100;
   private final String DELIMITER = ",";
+  private static int MAX_SIZE_NOTIFICATIONS = 20;
+  private static boolean isInitialized = false;
+
+  private int getMaxNotificationSize() {
+    if (isInitialized) return MAX_SIZE_NOTIFICATIONS;
+    try {
+      android.content.Context context =
```

Comment:
I don't like the fully qualified package names here vs imports alphabetic at the top and just the names here. Fully qualified package names when they are not critical to disambiguate similarly named objects do not match style